entries/m/mytax.illinois.gov.json

```
@@ -0,0 +1,18 @@
+{
+	"MyTax Illinois": {
+		"domain": "mytax.illinois.gov",
+		"img": "mytax.illinois.gov.png",
+		"tfa": [
+			"email",
+			"totp"
+		],
+		"documentation": "https://tax.illinois.gov/programs/mytax/authenticationguide.html",
+		"categories": [
+			"government",
+			"finance"
```

Only using the Government category would be best.
```suggestion
			"government"
```